docs/readmes/lte/extended_5G_sa_features.md

```diff
@@ -0,0 +1,173 @@
+    participant SessionD
+    participant PipelineD
+    participant PCF
+    AMF-->>SessionD: PDU Session initial request
+    note over SessionD: PDU Session with Rule 1
+    SessionD-->>AMF: SetSmfContext with Rule 1
+    note over SessionD: PDU Session Established
+    PCF-->>SessionD: PolicyReAuth for Session modification
+    note over PCF,SessionD: Rule 2 activate
+    note over SessionD: Store in RuleToProcess 
+    SessionD-->>AMF: SetSmfContext
+    note over SessionD,AMF: Rule 2 activate
+    AMF-->>UE: Request for Session modification
+    UE-->>AMF: Response for Session modification
+    AMF-->>SessionD: SetAmfSessionContext with Existing_PDU_Session
+    note over AMF,SessionD: Rule 2 activate
+    SessionD-->>PipelineD: Set SMF Sessions
+```
+
+Here, Rule 1 is the default QoS policy and Rule 2 is the QoS policy we are applying in Session Modification.
+
+Same sequence diagram is applicable for Deletion of QoS policy.
+
+### Limitations
+
+1. Currently AGW is supporting serial operation only, which means addition of a QoS policy or deletion of a QoS policy in a Session modification request. A single request containing add and delete is currently not supported.
+2. Right now this feature is still to be integrated with PCF. The testing has been using the stub CLIs in sessiond.
+
+## SUCI Extensions
+
+5G standards mitigates the vulnerabilities caused due to IMSI catchers by introducing SUCI. SUCI is a privacy-preserving identifier containing the concealed SUPI.
+
+In this release AGW supports two different profiles of [SUCI Extensions](https://docs.magmacore.org/docs/next/lte/suci_extensions).
+
+## Stateless feature
+
+The MME will store UE registrations/session management data in-memory. Any MME failure or restart will result in the loss of all data, including UE registration and Operator Account information, preventing UEs from accessing packet core services. As a result, UEs will need to reconnect and re-register.
+
+Stateless feature provides failure recovery mechanism by storing user data in a persistent storage.
+On occurrence of a failure, AGW will retrieve the UE data from the persistent storage, a partial service interruption may occur. However, once the AGW has been respawned then registered users can access the service without having a fresh registration.
+
+The MME will sync UE registration and session management data into persistent storage (Redis Database) for pre-defined trigger points. Incase of any MME failure/restart, MME will fetch UE specific data from Redis Database.
+
+![Stateless Feature](assets/lte/Stateless_feature.png?raw=true "Stateless Feature")
+
+## GTP extension header support
+
+The Extension Header Length field specifies the length of the particular Extension header in 4 octet units. The Next Extension Header Type field specifies the type of any Extension Header that may follow a particular Extension Header. If no such Header follows, then the value of the
+Next Extension Header Type shall be 0.

```

Comment:
- superflous line break.
- `shall be 0` sounds a bit ambiguous to me. What is meant? Should, must, can, will?

Reply:
Addressed Review Comment